deploy.sh

```diff
@@ -1,29 +1,29 @@
 #!/usr/bin/env bash
-set -e
+set -eu
 
-stack_name="$1"
-common_stack_name="${2:-kbv-hmrc-common-cri-api-local}"
```

Comment:
Can we keep this, need to able to specify a different common-stack, devs can deploy a common-stack different from `common-cri-api`

Reply:
Yes but the default needs to be empty then as the value in the script is incorrect so local deployments fail. The script on main doesn't work - this makes it work with the default common stack.

Reply:
And then deploy their dev stack to point to it 

`./deploy.sh ola-kbv-hmrc-cri ola-common-cri-api`

Reply:
OK do you mind if I do that in the next PR?

Reply:
Looks like that stack doesn't exist anymore so we can make the default empty and then the `common-cri-api ` will be used, or you'll be able to run the command you posted above to override it. We just don't want a bad default in the script